### ISSUE TITLE
Is there a way to get a 10 day forecast?

### ISSUE THREAD

**Issue by fury182** [chubin/wttr.in#682]
Is there a way to get a 10 day forecast? Or at least something more than 3 days?

Thanks

**Comment by chubin** [comment]
@fury182 No, at the moment it is not possible, because we have data only for three days, but I think it could be possible in future, if we would run a successful campaign on patreon, and gather enough funding to buy these data

**Comment by Angelk90** [comment]
@chubin : Is there anything new? 
I'd be interested to know if it were possible to get: 
The current weather + the forecast for the next `three` days instead of `two` as it does at the moment.

**Comment by romandas** [comment]
I am also interested in a 10-day forecast option. Is there a list somewhere of where wttr.in gets its data from?
